Documentation/CRDs/Block-Storage/ceph-block-pool-rados-namespace-crd.md

```diff
@@ -49,3 +49,34 @@ If any setting is unspecified, a suitable default will be used automatically.
 ### Spec
 
 - `blockPoolName`: The metadata name of the CephBlockPool CR where the rados namespace will be created.
+
+
+## Creating a Storage Class
+
+Once the RADOS namespace is created, an RBD-based StorageClass can be created to
+create PVs in this RADOS namespace. For this purpose, the `clusterID` value from the
+CephBlockPoolRadosNamespace status needs to be put into the `clusterID` field of the StorageClass
+spec. 
+
+Extract the clusterID from the CephBlockPoolRadosNamespace CR:
+
+ ```shell
+     $ kubectl -n rook-ceph  get cephblockpoolradosnamespace/namespace-a -o jsonpath='{.status.info.clusterID}'
+          80fc4f4bacc064be641633e6ed25ba7e
+     $
+```
+
+In this example, replace `namespace-a` by the actual name of the radosnamespace
+created before
```

Comment:
missing part of the sentence?

Reply:
@travisn  wrote: 
> missing part of the sentence?

sorry, what is missing or unclear here?